test/mocks/protobuf/mocks.h

```diff
@@ -13,6 +13,13 @@ class MockValidationVisitor : public ValidationVisitor {
   ~MockValidationVisitor() override;
 
   MOCK_METHOD(void, onUnknownField, (absl::string_view));
+
+  bool skipValidation() override { return skip_validation_; }
+
+  void setSkipValidation(bool s) { skip_validation_ = s; }
```

Comment:
For mocks, we generally don't have setters and expose the fiels, but this doesn't hurt I guess.